### DISCUSSION TITLE
QSV-preset hwaccel_args problems

### DISCUSSION THREAD

**Discussion by quenthal** [blakeblackshear/frigate#5741] [General]
Why do you think `hwaccel_args: -c:v h264_qsv` works, but `hwaccel_args: preset-intel-qsv-h264` causes cameras not working?

I have many different cameras (Dahua, Reolink etc.) running on intel 1240P with Iris XE on beta 9.

**Comment by NickM-27** [comment]
No idea without seeing the errors

**Reply by quenthal** [reply]
First, I have to apologize since after I wrote about this problem, I found few reported and closed issues that were nearly identical to my problem, for example https://github.com/blakeblackshear/frigate/issues/4921
 Using -c:v h264_qsv or preset-vaapi work fine, so there is probably no reason to study this more.

**Reply by NickM-27** [reply]
preset-vaapi is definitely recommended over qsv (IMO)
